app/config/mail.conf.dist.php

```diff
@@ -19,3 +19,12 @@
 $platform_email['SMTP_DEBUG'] = 0; // change to 1 to enable smtp debug
 $platform_email['SMTP_SECURE'] = 'tls'; // if you're using SSL: ssl; or TLS: tls. (only used if SMTP_AUTH==1)
 $platform_email['SMTP_UNIQUE_REPLY_TO'] = 0; // to use AWS SMS service, SMTP_UNIQUE_SENDER and this have to be = 1
+// DKIM requires the generation of a public/private keypair and the configuration of a TXT record in your DNS
+// The TXT record should look like this: chamilo._domainkey.yourdomain.ext IN TXT "v=1; k=rsa; s=chamilo; p=PubKey..."
+// to match the following selector
+// Also, using SMTP_UNIQUE_SENDER is required if users have e-mails from different domains
+$platform_email['DKIM'] = 0; //enable DKIM by setting this to 1
+$platform_email['DKIM_SELECTOR'] = 'chamilo'; // an indicator of the application sending the e-mail through this specific DKIM key
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 130 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5254203)